pyproject.toml

```diff
@@ -93,6 +93,7 @@ markupsafe = "^2.1.3"
 boto3 = "^1.28.63"
 numexpr = "^2.8.6"
 qianfan = "0.0.5"
+boto3 = "^1.28.63"
```

Comment:
I think all you gotta do is remove this and this should be good to go. 

Thanks for the PR

Reply:
resolved